docs/CONTRIBUTING.md

```diff
@@ -1,6 +1,19 @@
-As an open source project we welcome contributions of many forms. Please reach out before starting
-work on any major code changes. This will ensure we avoid duplicating work, or that
-your code can't be merged due to a rapidly changing code base. If you would like support
-for a library that is not listed, [contact support][1] to share a request.
+We welcome contributions of many forms to our open source project.
+However, please be aware of some of the policies below, and we *strongly* recommend reaching out before starting *any* code changes.
 
-[1]: https://docs.datadoghq.com/help
+## External Pull Request Policies
+
+Because of security policies in place, external pull requests have the following policies:
+
+- **Fork Required**: You **must** create pull requests from a fork of the repository. Only approved Datadog engineers have push access.
+- **Limited Testing Access**: External pull requests **cannot** run our full automated test suite.
+- **Merge Process**: Pull requests from forks **cannot be merged directly**. A `dd-trace-dotnet` contributor must create a new pull request after re-pushing your branch. Your commit attributes will remain.
```

Comment:
You _can_ merge the original PR though after doing this tho, just to be clear

Reply:
Merge it after we recreate the PR from a non-fork?

Reply:
Exactly e.g. for this PR: https://github.com/DataDog/dd-trace-dotnet/pull/6633

- They pushed to a fork
- I reviewed it
- Created a dd-trace-dotnet branch from their fork
- Did a "standard" (manual) run against that branch in Azure DevOps. 
  - Build statuses are associated to a _commit_, so the build statuses associated with my _manual_ run are associated with their PR.
- Once it all passes, you can merge the original PR directly.

It's not necessarily more convenient for us, but it means their PRs _are_ merged, rather than merging a duplicate PR. 🤷‍♂️

Reply:
ahh gotcha, I'll update the phrasing here to reflect that :)

Reply:
Okay, clarified that in the process. I didn't realize you could do that 😛